frontend/src/libs/components/input/input.tsx

```diff
@@ -41,7 +43,11 @@ const Input = <T extends FieldValues>({
 
   return (
     <label className={styles.label}>
-      <span className={getValidClassNames(styles.text, labelClassName)}>
+      <span
+        className={getValidClassNames(styles.text, labelClassName, {
+          [styles.required]: required,
+        })}
```

Comment:
Let's pass it this way
```suggestion
        className={getValidClassNames(styles.text, labelClassName, required && styles.required)}
```